### ISSUE TITLE
[Request] more AR biomes. (and mobs)

### ISSUE THREAD

**Issue by PercyJames** [Advanced-Rocketry/AdvancedRocketry#1933]
**What do you think is missing or what do you think can be done better**
in my opinion, i think one of the parts AR could expand on is the biomes. specifically, i think there could be more of them. i know of a group of addons that add biomes from KSP but, i think you could add more original biomes. i don't really have any suggestions for biomes however. i also think the mod could benefit from adding new alien mobs to put on the planets. 

basically i think you could expand on the planet building aspect of the mod.
